test/640-bridge-mtu.bats

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bats   -*- bats -*-
+#
+# bridge driver tests with explicit modes
+#
+
+load helpers
+
+function check_iface_mtu() {
+    local host_or_container=$1
+    local iface=$2
+    local mtu=$3
+
+    if [ "$host_or_container" = "host" ]; then
+        run_in_host_netns ip -j --details link show "$iface"
+    else
+        run_in_container_netns ip -j --details link show "$iface"
+    fi
```

Comment:
nit it should be possible to do `run_in_${host_or_container}_netns ip -j --details link show "$iface"`
so no need for the branch, but not blocking this is fine as well